miner_objects/dashboard.py

```diff
@@ -0,0 +1,58 @@
+import asyncio
+
+import bittensor as bt
+import uvicorn
+
+import template.protocol
+from fastapi import FastAPI, HTTPException
+
+class Dashboard:
+    def __init__(self, wallet, metagraph, config, is_testnet):
+        self.wallet = wallet
+        self.metagraph = metagraph
+        self.config = config
+        self.is_testnet = is_testnet
+
+        self.miner_data = {}
+
+        self.app = FastAPI()
+        self.setup_routes()
+
+    def setup_routes(self):
+        @self.app.get("/miner/{miner_id}")
+        async def get_miner_data(miner_id: str):
+            bt.logging.info("Dashboard stats request processing")
+            asyncio.run(self.get_stats_positions_from_validator())
+            if self.miner_data:
+                return self.miner_data
+            else:
+                raise HTTPException(status_code=404, detail="Miner not found")
+
+    def run(self, host="127.0.0.1", port=8000):
```

Comment:
wondering if we should choose a port less likely to have a conflict

Reply:
8080?